config/config-package.py

```diff
@@ -19,7 +19,7 @@
 Generated from:
 https://github.com/zopefoundation/meta/tree/master/config/{config_type}
 --> """
-FUTUTRE_PYTHON_VERSION = "3.11.0-alpha.7"
+FUTUTRE_PYTHON_VERSION = "3.11.0-alpha.6"
```

Comment:
```suggestion
FUTURE_PYTHON_VERSION = "3.11.0-alpha.6"
```

Reply:
Thanks for catching this, I am going to fix it in all places.